app/models/student_assessment.rb

```diff
@@ -9,34 +9,31 @@ class StudentAssessment < ActiveRecord::Base
 
   def valid_assessment_attributes
     case assessment.family
-    when 'MCAS'
-      # errors.add(:scale_score, "invalid attributes") unless valid_mcas_attributes
     when 'STAR'
-      # errors.add(:scale_score, "invalid attributes") unless valid_star_attributes
-    when 'DIBELS'
-      # errors.add(:scale_score, "invalid attributes") unless valid_dibels_attributes
+      errors.add(:percentile_rank, "invalid") unless valid_star_attributes?
+
+      # TODO: Add validation for STAR grade_equivalent field.
+      # This should be present for all STAR records, but because this wasn't
+      # backfilled to older records, ~40% of STAR records in production
+      # have grade_equivalent of nil.
```

Comment:
Not clear to me if the best practice is to add a validation even if it would mark ~40% of existing data as invalid, or just leave this note here and add in the validation once we've cleaned up the data. 

My instinct is to hold off on adding the validation for now. That makes all the other validations we're adding useful, because all things equal we expect our data to be in a valid state and any validation errors are from something unexpected.